STM32/DC/Core/Src/DCBoard.c

```diff
@@ -246,12 +221,13 @@ static void autoOff()
 
 static void handleButtonPress()
 {
-    // Button ports
-    GPIO_TypeDef *button_ports[] = { Btn_2_GPIO_Port, Btn_1_GPIO_Port,
-            Btn_4_GPIO_Port, Btn_3_GPIO_Port };
-    const uint16_t buttonPins[] = { Btn_2_Pin, Btn_1_Pin, Btn_4_Pin, Btn_3_Pin };
+    // Button ports - Button 1 and 2 position is swapped because... insert reason here?
+    GPIO_TypeDef *button_ports[] = { Btn_2_GPIO_Port, Btn_1_GPIO_Port, Btn_4_GPIO_Port, 
```

Comment:
Agree this swap looks like a bug / bad pin out implementation. The easiest thing is probably just renaming them in the .ioc.

Reply:
I will rename them in the IOC

Reply:
I just updated the code to have the order be from 1-6 which matches with the buttons.